model/VocabularyConfig.php

```diff
@@ -123,6 +124,46 @@ private function setPluginParameters(Easyrdf\Resource $pluginResource) : void
         }
     }
 
+    /**
+     * Sets array of configured property label overrides
+     *  @return void
+     */
+    private function setPropertyLabelOverrides() : void
+    {
+        $this->labelOverrides = array();
+        $overrides = $this->resource->allResources('skosmos:propertyLabelOverride');
+        if (!empty($overrides)) {
+            foreach ($overrides as $override) {
+                $this->setLabelOverride($override);
+            }
+        }
+    }
+
+    /**
+     * Updates array of label overrides by adding a new override from the configuration file
+     * @param Easyrdf\Resource $labelOverride
+     * @return void
+     */
+    private function setLabelOverride(Easyrdf\Resource $override) : void
+    {
+        $labelProperty = $override->getResource('skosmos:property');
+        $labelPropUri = $labelProperty->shorten();
+        if (empty($this->labelOverrides[$labelPropUri])) {
+            $this->labelOverrides[$labelPropUri]  = array();
+        }
+        $newOverrides = array();
+
+        $labels = $override->allLiterals('rdfs:label'); //property label overrides
+        foreach ($labels as $label) {
+            $newOverrides['label'][$label->getLang()] = $label->getValue();
+        }
+        $descriptions = $override->allLiterals('schema:description'); //optionally override property label tooltips
```

Comment:
I think it would be better to use `rdfs:comment` to define tooltip overrides. That would match with how property tooltips are read from vocabulary RDF data.

Reply:
Fixed in a new commit